tools/kmodules/sched_tp/sched_events.h

```diff
@@ -0,0 +1,134 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+#undef TRACE_SYSTEM
+#define TRACE_SYSTEM sched
+
+#if !defined(_SCHED_EVENTS_H) || defined(TRACE_HEADER_MULTI_READ)
+#define _SCHED_EVENTS_H
+
+#define PATH_SIZE		128
+#define SPAN_SIZE		128/4	/* assuming a max of 128 cpu system! */
```

Comment:
I thought about this but then one can override this value from command line if I remember correctly. It's not really a lot of memory. 128 bitmap is 2 words on 64bits systems and 4 on 32bits one

Reply:
Okay sod that I should have looked at the code again I already forgot it :D

Maybe I can make the events dynamic if the impact of this is really serious. The overutilized event is low traffic in general so I don't see that wasting too much space?

Reply:
Random thought: If these can be changed with a `-DCONFIG_NR_CPUS=42` when compiling the module, it's quite easy to fill-in the right value directly from LISA. It can default to a reasonable number for non-LISA users.